src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultRazorEditorFactoryService.cs

```diff
@@ -136,7 +135,7 @@ internal bool TryInitializeTextBuffer(ITextBuffer textBuffer)
             var parserFactory = razorLanguageServices.GetRequiredService<VisualStudioRazorParserFactory>();
             var braceSmartIndenterFactory = razorLanguageServices.GetRequiredService<BraceSmartIndenterFactory>();
 
-            var tracker = documentTrackerFactory.Create(textBuffer);
+            var tracker = documentTrackerFactory.Create(textBuffer)!;
```

Comment:
I guess this is the partner to the above comment. We're potentially storing a null, _and_ outputting it in a non-nullable parameter? Scary. Seems like `Create` is annotated incorrectly, or we need some `Assumes` at the least, so failures can actually be tracked down.

Reply:
> Seems like `Create` is annotated incorrectly

The default implementation of this method is:
https://github.com/dotnet/razor-tooling/blob/9b4b1b3ef61e2b5f62e76e2625762cc1cf81f11a/src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioDocumentTrackerFactory.cs#L83-L105
It is very clear, that it assumes a nullable return value.

> we need some `Assumes` at the least

Will do that